pie_docker.py

```diff
@@ -0,0 +1,27 @@
+"""
+Python3.6+ only
+"""
+from pie import *
+
+
+
+class Docker:
+    def __init__(self,options=[]):
```

Comment:
object instance as a parameter

Reply:
> object instance as a parameter

what do you mean? are you referring to the `class Docker(object):` thing in python 2 that makes classes "new style" (not needed in python 3)?

Reply:
https://stackoverflow.com/a/2831215

Reply:
Oh! that would have caught me out, I didn't realise.